grpc.go

```diff
@@ -62,6 +62,8 @@ var (
 
 	// set when StopTracing() is called.
 	stopTracingCalled = false
+
+	modprobePath = "/sbin/modprobe"
```

Comment:
Super dumb question. Is the path of `modprobe` the same for all distros? I know our images are today made with clear, but would this work if someone chooses to create their base image with a different distro?

Reply:
Not a dumb question at all and good to ask it!

In summary, "yes" since `modprobe(8)` is an admin command, so the binary must live in `/sbin` (which might of course be linked to `/usr/sbin`). Generally, hard-coding paths should be discouraged, but in this scenario, it's required to ensure we're calling the correct "official" binary (security).

Reply:
Thanks @jodh-intel